frontend/public/features.ts

```diff
@@ -104,11 +106,22 @@ const detectProjectsAvailable = dispatch => coFetchJSON(projectListPath)
       : handleError(err, FLAGS.PROJECTS_AVAILABLE, dispatch, detectProjectsAvailable)
   );
 
+  const projectRequestPath = `${k8sBasePath}/apis/project.openshift.io/v1/projectrequests`;
+  const detectCanCreateProject = dispatch => coFetchJSON(projectRequestPath)
+    .then(
+      res => setFlag(dispatch, FLAGS.CAN_CREATE_PROJECT, res.status === 'Success'),
+      err => {
+        setFlag(dispatch, FLAGS.CAN_CREATE_PROJECT, false)
+        dispatch(UIActions.setCreateProjectMessage(err.message));
```

Comment:
Is `message` always empty when there's no custom message? Do we need to check explicitly for 403?

I'm worried some other error (e.g. 429 Too Many Requests) would give a random error message that we show. We might retry if not 400, 403, 404, 500 as well.